src/components/menu/style/index.less

```diff
@@ -181,6 +181,10 @@
     }
   }
 
+  &-vertical {
```

Comment:
please add custom style in patch.less file, not the index.less since it will be sync with ant design react.